controllers.go

```diff
@@ -52,6 +52,26 @@ func imageController(o ServerOptions, operation Operation) func(http.ResponseWri
 	}
 }
 
+func determineAcceptMimeType(accept string) string {
+	for _, mime := range strings.Split(accept, ",") {
+		// strip off any other parameters after mime type
+		i := strings.Index(mime, ";")
```

Comment:
Consider using: https://golang.org/pkg/mime/#ParseMediaType

Reply:
Neat, didn't know about that. Thanks!